misc/vim/syntax/nit.vim

```diff
@@ -86,9 +86,9 @@ syn match  NITSharpBang	"\%^#!.*"
 syn match  NITComment	"#.*" contains=NITTodo,@Spell
 
 " Keywords
-syn keyword NITKeyword	 abstract intern new
-syn keyword NITDefine	 private public protected intrude readable writable redef
```

Comment:
Not `readable`. And if I keep `writable`, I think others like `noautoinit` should be added for consistency.